api/metrics_service.proto

```diff
@@ -39,7 +40,6 @@ message StreamMetricsMessage {
 
 // Configuration structure.
 message MetricsServiceConfig {
```

Comment:
Do we need this extra wrapper? If so, what other info should go into this message?

Reply:
I think this wrapper would be useful. For example, some of  things I was thinking are  
1. stats prefix that should be added before pushing metrics
2. should nodId be part of the metric name etc.
But I am not planning to implement any of these things in the initial iteration. Also do not have concrete use cases for them yet. So I think it is better to have this wrapper.

Reply:
I agree we should keep the wrapper in case we want to add more fields later as @ramaraochavali mentions.